tests/test_backend.py

```diff
@@ -356,11 +357,6 @@ def test_file(tmpdir, local_file, remote_file, version, ext, backend):
     backend.remove_file(remote_file, version, ext=ext)
     assert not backend.exists(remote_file, version, ext=ext)
 
-    if ext is None:
-        _, ext = os.path.splitext(local_file)
-    else:
-        ext = '.' + ext
```

Comment:
Why can you remove this part?

Reply:
It's left over when we were testing if the path returned by `put_file()` has the expected extension. But since we no longer return a path, these lines are also no longer needed. See https://github.com/audeering/audbackend/pull/47/files#diff-31e088e0e9d455d2164fd6251e7c23a77afc5561b3746cdf83e08d6a05142154L349